__tests__/app.test.js

```diff
@@ -87,6 +88,45 @@ describe("GET /api/articles/: article_id", () => {
     });
 });
 
+describe("GET /api/articles", () => {
+    test("status 200, responds with an array of article objects with the needed properties", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+            const expected = {
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(String)
+            }
+            expect(body.articles).toBeInstanceOf(Array);
+            expect(body.articles[0]).toMatchObject(expected);
+        });
```

Comment:
Instead of just checking the first article, I would use a forEach and check that every article returned has this same shape. The test is really good but if it checked every article it would be great. 

I would also have a length check in here to check how many articles you get back :)